Cargo.toml

```diff
@@ -10,6 +10,7 @@ cpal = "0.13.3"
 midir = "0.7.0"
 num-traits = "0.2.14"
 ringbuf = "0.2.6"
+rustfft = "6.0.1"
```

Comment:
It's probably worth looking a [the docs](https://docs.rs/rustfft/6.0.1/rustfft/) for this crate to get an idea of what the "planners" are for.